src/cromshell/utilities/HTTPUtils.py

```diff
@@ -0,0 +1,27 @@
+import logging
```

Comment:
The module name here should be a bit different - [see the PEP8 spec for details.](https://www.python.org/dev/peps/pep-0008/#package-and-module-names)

Reply:
done